.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+- repo: https://github.com/psf/black
+  rev: 22.10.0
+  hooks:
+  - id: black
+    args: ['--check', '--diff', '.']
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: v0.0.254
+  hooks:
+  - id: ruff
+    args: ['--ignore=E741,E501', '.']
```

Comment:
I'd put the ignored rules in `pyproject.toml` or `setup.cfg` so that IDEs and manual usages of `ruff` will use the same settings as the pre-commit hook.